### ISSUE TITLE
Extension fails with WSL on Windows 10

### ISSUE THREAD

**Issue by markfaine** [sydro/atom-ansible-vault#42]
Command failed: touch C:\... && echo "PASSWORD" > C:\...
'touch' is not recognized as an internal or external command,
operable program or batch file.

**Comment by sydro** [comment]
Hi, 
Did you test windows branch ( https://github.com/sydro/atom-ansible-vault/tree/windows-support ) ? Currently package doesn't support windows.

**Comment by markfaine** [comment]
No, didn't know there was a windows branch.  It's not exactly windows though, it's Ubuntu 18 WSL.  I guess I could just make it a feature request then?  I think that other extensions are implementing it using a flag 
"useWSL" or some such.
